benchcab/main.py

```diff
@@ -16,6 +17,13 @@ def parse_and_dispatch(parser):
         Parser object.
     """
     args = vars(parser.parse_args(sys.argv[1:] if sys.argv[1:] else ["-h"]))
+
+    # Intercept the verbosity flag to engage the logger
+    log_level = 'debug' if args.get('verbose', False) == True else 'info'
```

Comment:
This could be handled in the parser definition directly:

```
    args_subcommand.add_argument(
        "-v",
        "--verbose",
        help="Enable more detailed output in the command line.",
        action="store_const",
        default="info",
        const="debug",
        dest="log_level",
    )
```
or something like that because I haven't tried it out... I believe `action` isn't necessary but it doesn't hurt either.

Reply:
See, this was my first thought, but then I saw that the verbose argument is passed around 150+ times throughout the code so there is a bit of a refactoring exercise involved to ensure things still work. This suggestion is probably related to, but a bit outside the scope of this ticket.

Should we raise this as a different Issue and PR to address this?

Reply:
Yes, we can put it in a different issue.